src/main/java/duke/util/Utility.java

```diff
@@ -0,0 +1,29 @@
+package duke.util;
+
```

Comment:
Perhaps you can add some class header with a simple description of what this class is for.